Makefile

```diff
@@ -215,29 +216,48 @@ JUJUD_BIN_DIR              ?= ${BIN_DIR}
 OPERATOR_IMAGE_BUILD_SRC   ?= true
 
 # Import shell functions from make_functions.sh
+# For the k8s operator.
 BUILD_OPERATOR_IMAGE=sh -c '. "${PROJECT_DIR}/make_functions.sh"; build_operator_image "$$@"' build_operator_image
 OPERATOR_IMAGE_PATH=sh -c '. "${PROJECT_DIR}/make_functions.sh"; operator_image_path "$$@"' operator_image_path
 OPERATOR_IMAGE_RELEASE_PATH=sh -c '. "${PROJECT_DIR}/make_functions.sh"; operator_image_release_path "$$@"' operator_image_release_path
+# For the new k8s agent.
+BUILD_K8SAGENT_IMAGE=sh -c '. "${PROJECT_DIR}/make_functions.sh"; build_k8sagent_image "$$@"' build_k8sagent_image
```

Comment:
New variables and targets are defined for the new k8sagent stuff instead of refactoring existing ones by adding new toggle variables.
Because we may change a lot or even remove these new stuff.

Reply:
I'm wondering whether or not we want to just include the new k8s agent in our existing operator image, so we just have one image to deal with.  Is there a problem with including it in the current image?

Reply:
As discussed, we are going to keep k8sagent binary in a separate image for now.
We will merge it with the existing image if we want to in the future.